### PR TITLE
Update astro indexing recipe

### PR DESCRIPTION
**Pull request by jordangeizer**
Hi folks 👋

I went to implement the `Disable Admin UI Routes in Production` recipe last night and found it a bit hard to follow, having recently set up an Astro project with the Keystatic integration. Hope this clears it up a bit for any newcomers!

### CODE REVIEW

**Comment by simonswiss** [comment]
Oooohh good point — now that we rolled out the `keystatic()` integration this makes this guide much harder to comprehend!

@emmatown how hard would it be to allow to pass an option to the config that takes care of [short-circuiting the admin ui routes for production](https://github.com/Thinkmill/keystatic/pull/778/files#diff-305bdfda5cd916372b48362b979560bd0aa266314c7d51f4e6403601e5baf58dR58-R60), to avoid telling users to "eject" from the integration?

**Comment by emmatown** [comment]
People could just conditionally add the plugin? e.g.
```ts
plugins: [react(), markdoc(), ...(process.env.SKIP_KEYSATATIC ? [] : [keystatic()])]
```

**Comment by jordangeizer** [comment]
That works a charm, thank you @emmatown! Let me know if you'd like me to update this PR to that one liner @simonswiss!

**Comment by simonswiss** [comment]
@jordangeizer yes please, I think that's a much better approach ✨

Sorry about the delay.

**Review by emmatown — APPROVED** [review]
